tests/integration/cocoon/test_cocoon_portfolio_groups.py

```diff
@@ -0,0 +1,239 @@
+import unittest
+from pathlib import Path
+import pandas as pd
+import lusid
+import lusid.models as models
+from lusidtools import cocoon as cocoon
+from parameterized import parameterized
+from lusidtools import logger
+
+
+class CocoonTestPortfolioGroup(unittest.TestCase):
```

Comment:
I think we need some failure cases to ensure the unhappy path is easy for the user to deal with:

- portfolio not created when adding to group
- multiple instances of portfolio in source data